### PR TITLE
docs: README file update

### PR DESCRIPTION
**Pull request by rolsonquadras**
Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # edge-landing
-
+The project will have the static website contents to demonstrate [Hyperledger Aries Framework Go](https://github.com/hyperledger/aries-framework-go) features.
```
Comment (troyronda) [inline README.md]: I think just "content"

Comment (rolsonquadras) [reply README.md]: [Updated](https://github.com/trustbloc/edge-landing/compare/7999596ff8d1564ff9587d3f04a7395e9826b833..dca84235b5d11be2b87705dc7eb48f7be42102f3).

Comment (troyronda) [reply README.md]: sorry - and I think we mean to say static site generator

(see https://gridsome.org for terminology example)

Comment (rolsonquadras) [reply README.md]: [Updated](https://github.com/trustbloc/edge-landing/compare/dca84235b5d11be2b87705dc7eb48f7be42102f3..5af87131ed52ead0e798a66da9f0b9de1e49749e#diff-04c6e90faac2675aa89e2176d2eec7d8R2).

**Pull request merged**